README.md

```diff
@@ -72,7 +72,7 @@ The overall atmosphere of the image exudes tranquility, with the pelicans seemin
 | Cohere-command-r-plus | ❌ | ✅ | text | text |
 | Cohere-command-r-plus-08-2024 | ❌ | ✅ | text | text |
 | DeepSeek-R1 | ❌ | ❌ | text | text |
-| DeepSeek-V3 | ❌ | ❌ | text | text |
+| DeepSeek-R1-0528 | ❌ | ❌ | text | text |
```

Comment:
DeepSeek-V3 is gone?

Reply:
I removed the catalog filter and changed this PR to just be a model refresh which brough V3 back. It's still available in azureml, but is currently deprecated and set to retire in August